python/spear/game_world_service.py

```diff
@@ -92,11 +111,282 @@ def get_stable_name_for_actor_component(self, component, include_actor_name):
     def get_stable_name_for_scene_component(self, component, include_actor_name):
         return self._rpc_client.call("game_world_service.get_stable_name_for_scene_component", component, include_actor_name)
 
+    #
+    # Get actor and component tags
+    #
+
     def get_actor_tags(self, actor):
         return self._rpc_client.call("game_world_service.get_actor_tags", actor)
 
     def get_component_tags(self, actor):
         return self._rpc_client.call("game_world_service.get_component_tags", actor)
 
+    #
+    # Get class
+    #
+
     def get_class(self, uobject):
         return self._rpc_client.call("game_world_service.get_class", uobject)
+
+    #
+    # Create components
+    #
+
+    def create_component_on_actor(self, class_name, owner, name):
+        return self._rpc_client.call("game_world_service.create_component_outside_owner_constructor", class_name, owner, name)
+
+    def create_scene_component_on_actor(self, class_name, owner, name):
+        return self._rpc_client.call("game_world_service.create_scene_component_outside_owner_constructor_from_actor", class_name, owner, name)
+
+    def create_scene_component_on_component(self, class_name, owner, parent, name):
+        return self._rpc_client.call("game_world_service.create_scene_component_outside_owner_constructor_from_object", class_name, owner, parent, name)
+
+    def create_scene_component_on_owning_scene_component(self, class_name, owner, name):
+        return self._rpc_client.call("game_world_service.create_scene_component_outside_owner_constructor_from_component", class_name, owner, name)
+
+    #
+    # Create new object
+    #
+
+    def new_object(self, class_name, outer, name, flags, uobject_template, copy_transients_from_class_defaults, in_instance_graph):
+        return self._rpc_client.call("game_world_service.new_object", class_name, outer, name, {"ObjectFlags": json.dumps({"Enum": flags})}, uobject_template, copy_transients_from_class_defaults, in_instance_graph)
+
+    #
+    # Load object
+    #
+
+    def load_object(self, class_name, outer, name, filename, load_flags, sandbox, instancing_context):
+        return self._rpc_client.call("game_world_service.load_object", class_name, outer, name, filename, {"LoadFlags": json.dumps({"Enum": load_flags})}, sandbox, instancing_context)
+
+    #
+    # Get default object
+    #
+
+    def get_default_object(self, uclass, create_if_needed):
+        return self._rpc_client.call("game_world_service.get_default_object", uclass, create_if_needed)
+
+    #
+    # Get static class
+    #
+
+    def get_static_class(self, class_name):
+        return self._rpc_client.call("game_world_service.get_static_class", class_name)
+
+    #
+    # Spawn actor
+    #
+
+    def spawn_actor(self, class_name, location, rotation, spawn_parameters):
```

Comment:
Need `destroy_actor(actor)` immediately after `spawn_actor(...)` and `destroy_component(component, promote_children)` immediately after `create_scene_component_on_owning_scene_component(...)`.